src/actor.rs

```diff
@@ -0,0 +1,91 @@
+use crate::traits::{LoadableState, UptimeCheckerActor};
+use crate::types::{InitParams, MultiAddr, NodeInfo, PeerID};
+use crate::Error;
+
+const THRESHOLD_NUM: usize = 20000;
+const THRESHOLD_DUM: usize = 30000;
+
+pub struct Actor<S: LoadableState> {
```

Comment:
State needs to be `Serializable` so it can be persisted and accessed through the actor boundary/FVM: https://github.com/adlrocha/hc-subnet-actor/blob/53627c07b21d2ae96831a94a33f274b5d53e1bc0/src/state.rs#L31

Reply:
Yeah, I had it placed on the `struct`. Once I have the `hamt` issue figured out, I'll just use the struct instead of trait. If I placed Serializable + Deserizable in the trait it would complicate stuff. So I just kept it simple for now.